huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go

```diff
@@ -103,7 +104,7 @@ Value options: **PUB|SUB**, **PUB**, **SUB**, **DENY**.`,
 	}
 }
 
-func DmsRocketMQUserPermsRefSchema() *schema.Resource {
+func RocketMQUserPermsRefSchema() *schema.Resource {
```

Comment:
```suggestion
func rocketMQUserPermsRefSchema() *schema.Resource {
```
it seems it should be a private function

Reply:
done